composer.json

```diff
@@ -18,9 +18,9 @@
     "monolog/monolog": ">=1.13.1",
     "domnikl/statsd": ">=1.0.2",
     "psr/log" : ">=1.0.0",
-    "guzzlehttp/guzzle" : "^3.8",
     "firebase/php-jwt": "^3.0",
-    "doctrine/common": "^2.5"
+    "doctrine/common": "^2.5",
+    "guzzle/guzzle": "^3.8"
```

Comment:
> Where is the bit that changes the namespace?

@pads here - from `guzzlehttp` to `guzzle`, see e.g. the version guidance section [here](https://packagist.org/packages/guzzlehttp/guzzle)